types/custom-functions-runtime/index.d.ts

```diff
@@ -57,21 +52,12 @@ declare namespace CustomFunctions {
      */
     interface CancelableInvocation extends Invocation {
         /**
-         * @beta
          * Event handler called when the custom function is canceled.
          */
         onCanceled: () => void;
     }
 
     /**
-     * @beta
-     * @deprecated Use `CancelableInvocation` instead.
-     */
-    interface CancelableHandler extends CancelableInvocation {
```

Comment:
Just to mention, this is going to break users using this interface and `StreamingHandler`. Is that acceptable and should I merge it?

Reply:
This is OK. Users should use CancelableInvocation/StreamingInvocation instead.